metakb/models/__init__.py

```diff
@@ -0,0 +1 @@
+"""Data models for representing VICC records."""
```

Comment:
Nice to see the models declare separately from the harvesters  (puzzled how they relate to the dictionaries in the `_harvest_<>(singular)`  methods)